src/cpp/security/logging/LogTopic.h

```diff
@@ -18,16 +18,16 @@
 #ifndef _FASTDDS_RTPS_SECURITY_LOGGING_LOGTOPIC_H_
 #define _FASTDDS_RTPS_SECURITY_LOGGING_LOGTOPIC_H_
 
+#include <atomic>
+#include <fstream>
+#include <functional>
```

Comment:
Do we need this?

Reply:
No, but we need `<memory>`

Reply:
3759e24f9e1301257f406557872e7456ec2db0fd